aws-storage-s3/src/main/java/com/amplifyframework/storage/s3/service/AWSS3StorageService.java

```diff
@@ -262,4 +275,32 @@ private void startServiceIfNotAlreadyStarted() {
     public AmazonS3Client getClient() {
         return client;
     }
+
+    private Notification createDefaultNotification() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            createChannel();
+        }
+        int appIcon = context.getApplicationContext().getApplicationInfo().icon > 0 ?
```

Comment:
`context.getApplicationContext().getApplicationInfo().icon` will not look right in a lot of cases due to the way notifications are flattened into a solid color. Instead, I would propose providing a default icon named "amplify_storage_transfer_notification_icon", and then make it clear in our docs that the customer can provide a drawable under the same name to override our own. Ours would be a fallback if no override is provided.